compiler_gym/envs/llvm/llvm_benchmark.py

```diff
@@ -422,3 +423,80 @@ def _add_path(path: Path):
     timestamp = datetime.now().strftime("%Y%m%HT%H%M%S")
     uri = f"benchmark://user-v0/{timestamp}-{random.randrange(16**4):04x}"
     return Benchmark.from_file_contents(uri, bitcode)
+
+
+def split_benchmark_by_function(
+    benchmark: Benchmark, timeout_seconds: float = 300
+) -> List[Benchmark]:
+    """Split a benchmark into a list of benchmarks, one per function."""
+    original_uri = deepcopy(benchmark.uri)
+    original_bitcode = benchmark.proto.file.contents
```

Comment:
At least for me this needed to be `original_bitcode = benchmark.proto.program.contents`